src/preseed.json

```diff
@@ -0,0 +1,87 @@
+ {"platform": "windows8-64", "buildtype": "opt", "name": "tp5o", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "tp5o-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "svgr", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "svgr-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "chromez", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "chromez-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "other", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "other-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "g1", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "g1-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "g2", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "g2-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "dromaeojs", "expires": "", "action": "run"},
+ {"platform": "windows8-64", "buildtype": "opt", "name": "dromaeojs-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "tp5o", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "tp5o-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "svgr", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "svgr-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "chromez", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "chromez-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "other", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "other-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "g1", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "g1-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "g2", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "g2-e10s", "expires": "", "action": "coalesce"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "dromaeojs", "expires": "", "action": "run"},
+ {"platform": "osx-10-10", "buildtype": "opt", "name": "dromaeojs-e10s", "expires": "", "action": "coalesce"}
```

Comment:
Why do you need the "expires" keyword? It is blank and I don't see it used anywhere. May as well remove it?